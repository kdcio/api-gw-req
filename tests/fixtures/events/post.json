{
  "httpMethod": "POST",
  "headers": {
    "content-type": "application/json"
  },
  "body": "{\"message\":\"hello world\"}"
}
