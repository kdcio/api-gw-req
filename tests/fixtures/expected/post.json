{
  "path": "",
  "method": "POST",
  "query": {},
  "headers": {
    "content-type": "application/json"
  },
  "ip": "",
  "userAgent": "",
  "params": {},
  "body": { "message": "hello world" },
  "authorizer": {}
}
